j2lint/logger.py

```diff
@@ -3,17 +3,22 @@
 import logging
 from logging.handlers import RotatingFileHandler
 from logging import handlers
-
+import sys
 JINJA2_LOG_FILE = "jinja2-linter.log"
 
 logger = logging.getLogger('')
-logger.setLevel(logging.INFO)
-
 
-def add_handler(logger):
+def add_handler(logger, stream='', stream_level=''):
```

Comment:
add_handler(logger, streamHandler=False, log_level=logging.INFO)

rename stream to streamHandler seems more appropriate

Rename stream_level to log_level since it is a logger level rather than only for streamHandler